benchmarks/benchmarks/common.py

```diff
@@ -25,40 +25,90 @@
     'complex256',
 ]
 
+
+def memoize(func):
+    result = []
+    def wrapper(*a, **kw):
+        if not result:
+            result.append(func(*a, **kw))
+        return result[0]
+    return wrapper
```

Comment:
Or, you can click the `show regressions` button on the page, to show what benchmarks and when have regressed.